src/postamble.js

```diff
@@ -125,6 +125,10 @@ Module['then'] = function(func) {
 };
 #endif
 
+/**
```

Comment:
I run with `--js-opts 2 --llvm-opts 2 --minify 0`, so comments and whitespaces are all preserved while `#if` are processed already. Then I manually pass the generated code to closure compiler.